README.md

```diff
@@ -65,6 +96,21 @@ At this point you should be ready to go! Read on! :grin:
 
 ## Terms
 
+### **Hint**
+These are suggestions from the authors on directions you should think about
+or documentation you should look up. Generally speaking these are meant to be
+aids for when you're stuck on a problem.
+
+**NOTE:** Hints will always come with an assignment that they're supposed to be
+paired with. An example of this would be:
+
+If we've left a hint it's probably because a student has struggled at this
+point previously.
+
+```
```

Comment:
Should this block be above the last paragraph so it's more clear what the example is?